test/circle.cooltt

```diff
@@ -17,4 +17,4 @@ def loopn : nat -> Ω1s1 :=
     }
   ]
 
-#normalize {loopn 45}
+#normalize {loopn 5}
```

Comment:
:laughing:

Reply:
LOL

I wonder who can guess why I had to do this

Reply:
Well, you seemed to have removed all the optimizations that made 45+ possible...

Reply:
@favonia Yes precisely 😆 I couldn't understand some of them, so I just deleted them 🤦🏼

Reply:
But also, I think not only are the missing optimizations the cause, but the fact that I implemented the reachability check in a very very stupid way.